community/workgroups/index.md

```diff
@@ -22,7 +22,7 @@ permalink: /community/workgroups/index.html
           <p>{% t team.core_start %}</p>
           <h4>{% t team.contacts %}</h4>
           <ul class="logo">
-            <li>{% t team.chat %} <code>#monero-dev</code> <a class="chats-img" href="irc://chat.freenode.net/#monero-dev"><img class="freenode" src="/img/freenode.png" title="freenode" alt="freenode logo"></a> <a class="chats-img" href="https://matrix.to/#/#freenode_#monero-dev:matrix.org"><img class="matrix" src="/img/matrix-logo.svg" title="Matrix" alt="Matrix logo"></a> <a class="chats-img" href="https://mattermost.getmonero.org/monero/channels/monero-dev"><img class="mattermost" src="/img/mattermost.png" title="MatterMost" alt="Mattermost logo"></a></li>
+            <li>{% t team.chat %} <code>#monero-dev</code> <a class="chats-img" href="irc://irc.libera.chat/#monero-dev"><img class="libera" src="/img/libera.svg" title="libera" alt="libera logo"></a> <a class="chats-img" href="https://matrix.to/#/#libera_#monero-dev:matrix.org"><img class="matrix" src="/img/matrix-logo.svg" title="Matrix" alt="Matrix logo"></a> <a class="chats-img" href="https://mattermost.getmonero.org/monero/channels/monero-dev"><img class="mattermost" src="/img/mattermost.png" title="MatterMost" alt="Mattermost logo"></a></li>
```

Comment:
The Libera matrix format looks different.

```
03:11 <Lizzie[m]> There wasn't announcement from Matrix yet afaik, but there have been from Libera if I'm not mistaken. IRC channels can be joined through e.g. #monero:libera.chat
```

Reply:
`#monero:libera.chat` is to directly join libera rooms from matrix. We shouldn't use that, but the address of the matrix room instead (which will be then bridged to libera). Fixing.